NOTICE

```diff
@@ -45,7 +46,7 @@ In addition to using the Software, we encourage the community to join the Softwa
 
 ==========================================
 
-To join the ESMValTool Development Team, please contact Prof. Veronika Eyring (veronika.eyring@dlr.de) and Dr. Axel Lauer (axel.lauer@dlr.de).
+To join the ESMValTool Development Team, please contact Dr. Birgit Hassler (veronika.eyring@dlr.de) and Dr. Axel Lauer (axel.lauer@dlr.de).
 
```

Comment:
I think we will have to use also my email address, not just my name...  ;)
So change "Dr. Birgit Hassler (veronika.eyring@dlr.de)" to "Dr. Birgit Hassler (birgit.hassler@dlr.de)".